webapp/components/admin_console/users_and_teams_settings.jsx

```diff
@@ -25,9 +26,14 @@ export default class UsersAndTeamsSettings extends AdminSettings {
     }
 
     getConfigFromState(config) {
+        let maxUsersPerTeam = this.parseIntNonZero(this.state.maxUsersPerTeam);
+        if (maxUsersPerTeam === 1) {
```

Comment:
I don't think we should prevent the user from entering 1 user per team (as dumb as that sounds). Maybe change it to this instead
```
let maxUsersPerTeam = this.parseInt(this.state.maxUsersPerTeam);
if (maxUsersPerTeam === 0) {
```